CHANGELOG.md

```diff
@@ -21,7 +21,9 @@ Changelog
 * restructure packages and classes within the controller and output packages ([#117])
 * extend docs on contribution types ([#134])
 * round coordinates of returned OSM features to 7 decimal places ([#138])
+* wrap all lines longer than 100 characters ([#83])
```

Comment:
maybe to make it clear that it's about the code lines (and not the output json/csv lines for example):

```suggestion
* improve code style by wrapping all code lines longer than 100 characters ([#83])
```